### ISSUE TITLE
Bug: OCHRE asks for a cooling setpoint schedule when a ResStock building has no cooling

### ISSUE THREAD

**Issue by JingWang-CUB** [NREL/OCHRE#135]
I hit the exception error below when running ResStock models in OCHRE. This building has no cooling system so the hpxml file doesn't come with a weekday/weekend cooling setpoint schedule, but OCHRE is requiring it somehow. 
![image](https://github.com/user-attachments/assets/2295683b-f546-48b6-a844-c77c1db26c5c)
![image](https://github.com/user-attachments/assets/f6fc8670-b02d-45ed-bea5-7f7637516d01)

**Comment by mnblonsky** [comment]
Are you specifying HVAC Cooling (or an ASHP or MSHP) in the `dwelling_args`? That's the only thing I can think of why this would happen.

**Comment by JingWang-CUB** [comment]
Right, we added the following in `dwelling_args` to force realistic behavior of HVAC and WH. If this has caused this issue, is there an easy fix for it?
![image](https://github.com/user-attachments/assets/9fd4c799-7e1a-409f-8d9b-483a1b18300b)

**Comment by mnblonsky** [comment]
Yeah, I would remove the 'HVAC Cooling' line. That should solve it. I guess it's a little tricky since you have to know in advance whether there's cooling or not. There's probably not a great way to know that unless you read the HPXML file first.

**Comment by JingWang-CUB** [comment]
I can read the HPXML file and determine if the building has cooling or not, or I can read the buildstock.csv. @jmaguire1 , thoughts?

**Comment by jmaguire1** [comment]
@JingWang-CUB: Do you still need to parse in those arguments in the first place now that we decided on 2 minute timestep? We were doing that specifically to support 5 minute timesteps not defaulting to realistic controls, weren't we?

IF you do have to parse in whether cooling is present or not, I'd just do whatever's easier. Using HPXML is more flexible, but if you're already loading the buildstock.csv file then go with that.

**Comment by JingWang-CUB** [comment]
Removing the following arguments solved this issue. 
![image](https://github.com/user-attachments/assets/6e9f14f7-10ef-4726-b602-cda3cce34b60)
